server/routes/changeEstablishment.ts

```diff
@@ -1,18 +1,18 @@
 import type { RequestHandler, Router } from 'express'
+import { Prison } from '../@types/bapv'
 
 import asyncMiddleware from '../middleware/asyncMiddleware'
 
 export default function routes(router: Router): Router {
   const get = (path: string | string[], handler: RequestHandler) => router.get(path, asyncMiddleware(handler))
 
-  const enabledPrisons = [
-    { prisonId: 'HEI', name: 'Hewell (HMP)' },
-    { prisonId: 'BLI', name: 'Bristol (HMP)' },
+  const enabledPrisons: Prison[] = [
+    { prisonId: 'HEI', prisonName: 'Hewell (HMP)' },
+    { prisonId: 'BLI', prisonName: 'Bristol (HMP)' },
   ]
 
   get('/', (req, res, next) => {
     res.render('pages/changeEstablishment', {
-      hidePhaseBanner: true,
```

Comment:
Just checking this was meant as part of this PR?

Reply:
Well spotted - yes, that was intentional as the change establishment page is supposed to have the phase banner 🙂